### DISCUSSION TITLE
S57B

### DISCUSSION THREAD

**Discussion by CAP1Sup** [CAP1Sup/Intellistep#36] [General]
This is for all discussion related to the S57B and its support of the Intellistep project

**Comment by CAP1Sup** [comment]
@bilalibrir could you post progress on your schematic in here (in the thread under this comment)? As far as a quick glance from me, all of the pins and chips seem to be the same. Did BTT really just copy a board and rename it? I'd love to add support for the S57B, so a good schematic would be terrific!

**Reply by bilalibrir** [reply]
this schematics is reverse-engineered from the BTT S57B V2.0 Board.
I don't have a real module, all the work is done from images I got from the net.
if you could share some macro photos of BTT board with visible values of smd resistors and dc / dc buck converter. I will update the schematics as soon as possible.
my plane is to make an open source hardware version using the blue pill board that share the same pin map with the BTT S57B v2.0 board that share the same software.
![BTT_S57B_v2 0 (ver 1)]
note: (no usb, pin used for CAN), (the BTT S57B v2.0 use stm32f103cBt6 and the bluepill use stm32f103c8t6)
(https://user-images.githubusercontent.com/86444589/123391615-1dfe1900-d594-11eb-81da-1baa10a93478.png)
